workspace.json

```diff
@@ -3,11 +3,12 @@
         "apps/Demo"
     ],
 
-    "frameworks": {        
-        "ext": {
+    "frameworks": {        "ext": {
```

Comment:
Why the strange formatting?

Reply:
Yes I missed  to notice. Closing this PR and open new PR